src/Microsoft.ML.Data/Properties/AssemblyInfo.cs

```diff
@@ -13,6 +13,8 @@
 [assembly: InternalsVisibleTo(assemblyName: "Microsoft.ML.Predictor.Tests" + PublicKey.TestValue)]
 [assembly: InternalsVisibleTo(assemblyName: "Microsoft.ML.TimeSeries.Tests" + PublicKey.TestValue)]
 [assembly: InternalsVisibleTo(assemblyName: "Microsoft.ML.Benchmarks" + PublicKey.TestValue)]
+[assembly: InternalsVisibleTo(assemblyName: "Microsoft.ML.StaticPipelineTesting" + PublicKey.TestValue)]
```

Comment:
`Microsoft.ML.StaticPipelineTesting` is old and has been removed. You shouldn't be adding it back here.

Reply:
done